chromeos/chromeos.dev
# 🆕 Update ChromeOS spaces and logo

* Updates space automation to go from "Chrome OS" to "ChromeOS"
* Fix (and activate) tests for space cleanup
* Ensure space automation works on attributes as well as content
* Update logo ball
* Update favicon and PWA icons with new logo ball
* Dependency updates
